src/base/DebugMessage.ml

```diff
@@ -25,24 +25,24 @@ let plog msg =
   if get_debug_level () <> Debug_None then
   let fname = get_log_file () in
   Out_channel.with_file fname ~append:true
-    ~f:(fun h -> Out_channel.output_string h msg)
+    ~f:(fun h -> Out_channel.(output_string h msg; flush h))
```

Comment:
I'm not sure about this change, but I can't get the logger to output anything during the run of `scilla-server` without `flush`'ing the output channel...